client/src/components/Auth/Signin.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from "react";
+// import { Redirect } from "react-router-dom";
+
+export default class Signin extends Component {
+  // Added state here for use latter when adding authentication
+  constructor(props) {
+    super(props);
+    this.state = {
+      authenticated: false,
+      password: "",
+      username: "",
+      email: ""
+    };
+  }
+  // This is to check for authenticated when we add it in latter this code may change
+  checkAUthenticated = () => {
```

Comment:
I believe the casing should be updated to checkA**u**thenticated.

Reply:
Seconded.